CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/FunctionRequestHandler.swift

```diff
@@ -0,0 +1,181 @@
+//
```

Comment:
We can remove this too as we are not going to test functions. They are not tested on CBS too.

Reply:
I added it as a part of adding all the APIs. I'll leave it there in case it might be needed in the future.